packages/safe-core-sdk-types/src/types.ts

```diff
@@ -12,7 +12,7 @@ export enum OperationType {
 export interface MetaTransactionData {
   readonly to: string
   readonly value: string
-  readonly data: string
+  readonly data?: string
```

Comment:
Is this change purely for the scripts? I think we'll have to tweak the frontend in a few places because of this (not an issue though).

Reply:
I undid this change in favor of this better solution: https://github.com/safe-global/safe-core-sdk/pull/307